packages/components/src/Typography/css/TextColors.css

```diff
@@ -50,34 +54,218 @@
   color: var(--color-text--secondary);
 }
 
+.textReverse {
```

Comment:
Boggles my mind that we didn't have these colors with "text" in their name already in here 😅